osu.Game/Overlays/Comments/CommentEditor.cs

```diff
@@ -134,19 +147,27 @@ private void load(OverlayColourProvider colourProvider)
             });
 
             TextBox.OnCommit += (_, _) => commitButton.TriggerClick();
+            User.BindTo(api.LocalUser);
         }
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
             Current.BindValueChanged(_ => updateCommitButtonState(), true);
+            User.BindValueChanged(_ => updateTextBoxState());
```

Comment:
Not sure why this isn't `BindValueChanged(..., true)`? What if the user is not initially logged in?

Reply:
In truth, I didn't understand what `true` there did. I duplicated the login in `updateTextBoxState` and in the textbox drawable itself:

```csharp
TextBox = new EditorTextBox
{
    ...
    ReadOnly = !API.IsLoggedIn
}
```

I'm going to assume that leaving off `ReadOnly` in the drawable and using `runOnceImmediately` is the better approach.

Reply:
> I'm going to assume that leaving off `ReadOnly` in the drawable and using `runOnceImmediately` is the better approach.

Is my general preference, yeah.